pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -75,6 +75,12 @@ metadata:
 {{- end }}
 {{- end }}
 spec:
+{{- if .IPFamilies }}
+  ipFamilies: [ {{ .IPFamilies }} ]
```

Comment:
never want to set both?

Reply:
Both are optional fields. Let the user to decide what to set for the testing.

Reply:
I mean do we not ever want to test `ipFamilies: [ipv4, ipv6]`?

Reply:
We test those. Here is an example:
```
var deployConfig = map[string][3]string{
        "ipv4":      {"echo-ipv4", "IPv4", "SingleStack"},
        "ipv6":      {"echo-ipv6", "IPv6", "SingleStack"},
        "dual-ipv6": {"echo-dual-ipv6", "IPv6, IPv4", "RequireDualStack"},
        "dual-ipv4": {"echo-dual-ipv4", "IPv4, IPv6", "RequireDualStack"},
}
```
```
echo.Config{
          Service:            deployConfig[name][0],
          Namespace:     app.Namespace,
          Ports:                EchoPorts,
          Subsets:           []echo.SubsetConfig{{}},
          WorkloadOnlyPorts: WorkloadPorts,
          IPFamilies:        deployConfig[name][1],
          IPFamilyPolicy:    deployConfig[name][2],
}
```

Reply:
I would make `IPFamilies` a `[]string` instead of hacking it in with a `string` with a comma then